lte/gateway/c/oai/lib/mobility_client/MobilityClientAPI.cpp

```diff
@@ -43,7 +43,7 @@ using magma::lte::MobilityServiceClient;
 extern task_zmq_ctx_t spgw_app_task_zmq_ctx;
 
 static itti_sgi_create_end_point_response_t handle_allocate_ipv4_address_status(
-    const grpc::Status& status, struct in_addr inaddr, const char* imsi,
+    const grpc::Status& status, int vlan, struct in_addr inaddr, const char* imsi,
```

Comment:
why `vlan` comes as one of the first parameters? Placing it in front of other parameters creates the impression that it has higher importance than the other parameters. I suggest moving it post `pdn_type` and before `sgi_create_endpoint_resp`.

Reply:
I am just following L2, L3 layer parameter. IP address has to be coupled with vlan, without vlan context IP address does not identify a uniq host. I am fine with parameter ordering change. are you ok moving both IP address and vlan after PDN type.

Reply:
done.